sdk/communication/communication-chat/CHANGELOG.md

```diff
@@ -1,11 +1,17 @@
 # Release History
 
-## 1.0.0-beta.4 (Unreleased)
+## 1.0.0-beta.4 (2021-02-09)
 
 ### Breaking Changes
 
 - Replaced `CommunicationUser` with `CommunicationUserIdentifier`.
 - Replaced `CommunicationUserCredential` with `CommunicationTokenCredential`.
+- Support for CreateChatThreadResult and AddChatParticipantsResult to handle partial errors in batch calls.
+- Added idempotency identifier parameter for chat creation calls.
+- Added support for readreceipts and getparticipants pagination.
```

Comment:
That's not what the functions are called, right? It's `listParticipants` for example